API change in SIP 4.8

--- origsrc/qzion-mainline/python/qzion/qzionobject.sip	2009-02-18 15:38:44.000000000 -0600
+++ src/qzion-mainline/python/qzion/qzionobject.sip	2009-06-23 13:20:16.166117000 -0500
@@ -10,7 +10,7 @@ class QZionObject : QObject
 
 %ConvertToSubClassCode
     static struct classes {
-        char *name;
+        const char *name;
         sipWrapperType **type;
     } cls[] = {
         {sipName_QZionText, &sipClass_QZionText},
